src/getting-started.md

```diff
@@ -131,204 +124,188 @@ are two major downsides:
-“Country” and “Capital City”.
-
-When Anki checks for duplicates, it only compares other notes of the
-same type. Thus if you add a capital city called “Orange” using the
-capital city note type, you won’t see a duplicate message when it comes
-time to learn how to say “orange” in French.
+"Country" and "Capital City".
 
-When you create a new collection, Anki automatically adds some standard
-note types to it. These note types are provided to make Anki easier for
-new users, but in the long run it’s recommended you define your own note
-types for the content you are learning. The standard note types are as
-follows:
+Anki comes with some standard
+note types included. These note types are provided to make Anki easier for
+new users, but in the long run it's recommended you create your own note
+types specifically for the content you are learning. The standard note types are:
 
 - **Basic**\
-  Has Front and Back fields, and will create one card. Text you enter in
-  Front will appear on the front of the card, and text you enter in Back
+  Has "Front" and "Back" fields, and will create one card. Text you enter in
+  "Front" will appear on the front of the card, and text you enter in "Back"
   will appear on the back of the card.
 
 - **Basic (and reversed card)**\
-  Like Basic, but creates two cards for the text you enter: one from
-  front→back and one from back→front.
+  Like "Basic", but creates two cards for the text you enter: 
+  front→back and back→front.
 
 - **Basic (optional reversed card)**\
-  This is a front→back card, and optionally a back→front card. To do this,
-  it has a third field called “Add Reverse.” If you enter any text into
-  that field, a reverse card will be created. More information about this
-  is available in the [Cards and Templates](templates/intro.md) section.
+  Like "Basic", but has a third field called "Add Reverse". If you enter any text into
+  that field, a reversed card (back→front) will also be created. For details, see the [Cards and Templates](templates/intro.md) section.
 
 - **Basic (type in the answer)**\
-  This is essentially Basic, with an extra text box on the front where you
-  can type your answer in, after flipping to the back your input would be
-  checked and compared with the answer. More information is available in the
+  This is essentially "Basic", with an extra text box on the front where you
+  can type your answer in. After flipping to the back, you can check and
+  compare your input with the actual answer. For details, see the
   [Checking Your Answer](templates/fields.md#checking-your-answer) section.
 
 - **Cloze**\
-  A note type which makes it easy to select text and turn it into a cloze
-  deletion (e.g., “Man landed on the moon in \[…​\]” → “Man landed on the
-  moon in 1969”). More information is available in the [cloze deletion](editing.md#cloze-deletion) section.
+  A note type which makes it easier to select text and turn it into a cloze
```

Comment:
```suggestion
  A note type that makes it easy to select text and turn it into a cloze
```
"Easier" sounds wrong, because we're not comparing to anything -- it's either possible (with this note type), or impossible (without it). If you don't want to say "easy," perhaps -- 

```suggestion
  A note type that allows you to select text and turn it into a cloze
```
And switching from "which" to "that" because it's necessary clause.

Reply:
This one is hidden, but not resolved.